src/ui/ui.rs

```diff
@@ -671,11 +674,12 @@ fn handle_redraw_event(
                         grid.get_rect_for_cell(popupmenu.row, popupmenu.col);
 
                     state.popupmenu.set_anchor(rect);
-                    state.popupmenu.show();
                     state
                         .popupmenu
                         .select(popupmenu.selected as i32, &state.hl_defs);
 
+                    state.popupmenu.show();
```

Comment:
Moved this because of the info in [this comment](https://github.com/vhakulinen/gnvim/blob/68e3fc8d2e8f6f7bbdd73641b00031c16004e8cc/src/ui/popupmenu/lazy_loader.rs#L134-L135) (since there may be a callback attached by `popupmenu.show()` when it is called now; see [`show_menu_for_all_items`](https://github.com/smolck/gnvim/blob/94cb9d476fa5bd198bbf032ecf110e029ff65d15/src/ui/popupmenu/popupmenu.rs#L281) which may be called [here](https://github.com/smolck/gnvim/blob/94cb9d476fa5bd198bbf032ecf110e029ff65d15/src/ui/popupmenu/popupmenu.rs#L368) in the `popupmenu.show` function):

> Only one callback can exists at a time (e.g. when we are loading items).

Is what I've done here okay, or is there a better way? What I've done works, but it feels a bit off to me.